fs_base_multi_image/tests/test_base_multi_image.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import base64
+import io
+import os
+import tempfile
+
+from odoo_test_helper import FakeModelLoader
+from PIL import Image
+
+from odoo.addons.base.tests.common import BaseCommon
+from odoo.addons.fs_image.fields import FSImageValue
+
+
```

Comment:
```suggestion

@odoo.tests.tagged("post_install", "-at_install")
```
Hi @rousseldenis, can you add the following code? Using the `BaseCommon` class and not moving it to `post_install` could result in tests of modules that depend on this PR failing like this: https://github.com/xaviedoanhduy/storage/actions/runs/14566155548/job/40855697358